src/katello/agent/katelloplugin.py

```diff
@@ -114,11 +114,10 @@ def setup_plugin():
     if not ConsumerIdentity.existsAndValid():
         # not registered
         return
-    cfg = plugin.cfg()
     rhsm_conf = Config(RHSM_CONFIG_PATH)
     certificate = ConsumerIdentity.read()
-    cfg.messaging.url = 'ssl://%s:5671' % rhsm_conf['server']['hostname']
-    cfg.messaging.uuid = 'pulp.agent.%s' % certificate.getConsumerId()
+    plugin.cfg.messaging.url = 'amqps://%s' % rhsm_conf['server']['hostname']
```

Comment:
Can we default to 5467 here? The existing amqps port is in use by qpidd, and we decided not to change that.